test/comparenumbers.test.js

```diff
@@ -0,0 +1,49 @@
+// IMPORT MODULES under test here:
+import compareFunction from '../javafunction/comparenumbersfunction.js';
+// import example from '../src/example.js';
+
+const test = QUnit.test;
+
+test('4, 4', function(assert) {
```

Comment:
The test description should be a string that describes what you are testing in words. For example, 'correct guess'.